src/polygon.cpp

```diff
@@ -936,16 +940,21 @@ namespace manifold {
  * references back to the original vertices.
  * @param epsilon The value of &epsilon;, bounding the uncertainty of the
  * input.
+ * @param allowConvex If true (default), the triangulator will use a fast
```

Comment:
Where do we use this?

Reply:
This was how I fixed Simplify() - the surface looks like garbage with convex triangulation; ear clipping sorts by quality so it does quite well. Figured others might need it too, so probably best in the public API anyway.